spring-cloud-stream/src/main/java/org/springframework/cloud/stream/binder/AbstractMessageChannelBinder.java

```diff
@@ -402,6 +422,7 @@ protected void afterUnbindProducer(ProducerDestination destination,
 			if (properties.isAutoStartup() && consumerEndpoint instanceof Lifecycle) {
 				((Lifecycle) consumerEndpoint).start();
 			}
+			this.consumerCustomizer.configure(consumerEndpoint, name, group);
```

Comment:
@garyrussell ,

Sorry, I don't notice this because everything else is OK.
So, I think this is already too late to customize it after `afterPropertiesSet()` and `start()`.

Please, fix it in subsequent commit.

Reply:
Doh